lib/src/component_declaration/disposable_manager_proxy.dart

```diff
@@ -63,21 +63,18 @@ mixin DisposableManagerProxy on react.Component implements DisposableManagerV7 {
       _getDisposableProxy().listenToStream(stream, onData,
           onError: onError, onDone: onDone, cancelOnError: cancelOnError);
 
-  @override
```

Comment:
These weren't actually overrides and were infos in Dart 2, but got upgraded to warnings in Dart 3.